docker-compose.yml

```diff
@@ -29,9 +29,6 @@ services:
         hard: -1
     volumes:
       - esdata1:/usr/share/elasticsearch/data
-    # TODO: close these ports (fititnt, 2018-04-14 10:22 BRT)
-    ports:
-      - 9200:9200
```

Comment:
What do you think about make both MongoDB and ElasticSearch annotation on exposed ports consistent?

For example, if both are just commented, both have an explanation for a user that they can uncomment these lines to temporary expose these ports and debug with external tools.